indico/modules/events/papers/models/files.py

```diff
@@ -49,11 +50,11 @@ class PaperFile(StoredFileMixin, db.Model):
         nullable=True
     )
 
-    contribution = db.relationship(
+    _contribution = db.relationship(
```

Comment:
Why can't `contribution` be a "first-class" attribute of `PaperFile`? We've also got `subcontrib.event_new == subcontrib.contribution.event_new`... I still don't see any problem with that.